tests/run/java-lang-integer/src/main/scala/IntegerTest.scala

```diff
@@ -0,0 +1,18 @@
+import java.lang.Integer
+
+object IntegerTest {
+  def main(args: Array[String]): Unit = {
+    testIntegerToString()
+  }
+
+  def testIntegerToString() = {
+    // issue #270
+    println(Integer.toString(1).equals("1"))
+    println(Integer.toString(-1).equals("1"))
+    println(Integer.toString(123).equals("123"))
+    println(Integer.toString(-123).equals("-123"))
+    println(Integer.toString(1234).equals("1234"))
+    println(Integer.toString(-1234).equals("-1234"))

```

Comment:
I've just realised that those should have been `assert`s not `println`s. @fbertra, can you please submit a fix for this?